### DISCUSSION TITLE
Incorrect API Documentation for creating a new deployment

### DISCUSSION THREAD

**Discussion by xuanyu66** [vercel/community#4502] [Bug Report]
### Page to Investigate

https://vercel.com/docs/rest-api/endpoints#create-a-new-deployment

### Steps to Reproduce

https://vercel.com/docs/rest-api/endpoints#create-a-new-deployment contains several errors. Firstly, there are duplicated query parameters, both named 'forceNew.' Secondly, many of the required parameters appear to be optional to deployment, such as `databases`. Thirdly, many of the required parameters appear to be unrelated to deployment, such as `branch_logical_size_limit`.

**Comment by AndSteShape** [comment]
I have noticed the same issue. Really need this endpoint to be documented correctly, as the application I am building depends on it.
